js/utils/constants.js

```diff
@@ -918,6 +918,115 @@
        */
       VC_TAG_FOUND: 'videoTagFound',
 
+      /**
+       * This event is triggered when an ad sdk has been loaded successfully. the handler is called with:
```

Comment:
The 't' in second sentence needs to be capitialized.

Reply:
Fixed